pcsx2/MTVU.cpp

```diff
@@ -458,10 +459,17 @@ void VU_Thread::ExecuteVU(u32 vu_addr, u32 vif_top, u32 vif_itop, u32 fbrst)
 	CommitWritePos();
 	gifUnit.TransferGSPacketData(GIF_TRANS_MTVU, NULL, 0);
 	KickStart();
-	u32 cycles = std::min(Get_vuCycles(), 3000u);
-	cpuRegs.cycle += cycles * EmuConfig.Speedhacks.EECycleSkip;
-	VU0.cycle += cycles * EmuConfig.Speedhacks.EECycleSkip;
+	u32 cycles = std::max(Get_vuCycles(), 4u);
+	u32 skip_cycles = std::min(cycles, 3000u);
+	cpuRegs.cycle += skip_cycles * EmuConfig.Speedhacks.EECycleSkip;
+	VU0.cycle += skip_cycles * EmuConfig.Speedhacks.EECycleSkip;
```

Comment:
Is the behavior change to MTVU+Instant VU1 here if `Get_vuCycles()` returns a number less than 4 intentional?

Reply:
Yes, it's to stop it returning 0, which would break the internal interrupt stuff

Reply:
So this causes problems with the current code?

Reply:
It would do had I not done that, yeah.

Reply:
So it looks like the interrupt is only when Instant VU1 is off, not on, which leads me back to: was it intentional to change the behavior when Instant VU1 is on?

Reply:
It isn't, but it really won't make any difference, 4 cycles is practically zero

Reply:
Further to this it only matters of you have cycle skipping enabled